.travis.yml

```diff
@@ -31,7 +31,9 @@ matrix:
             - build-essential
             - libgtk-3-dev
             - libxtst-dev
-            - libavcodec57
+            - libavcodec-dev
```

Comment:
These libraries probably should be included as well in readme/docs, as these may not be installed by default by some Linux distributions (in follow up PRs)